q2_shogun/_shogun.py

```diff
@@ -65,7 +65,8 @@ def nobunaga(query: QIIME1DemuxFormat, reference_reads: DNAFASTAFormat,
                            database, reference_reads, reference_taxonomy)
 
         # run aligner
-        cmd = ['shogun', 'align', '-i', str(query), '-d', tmpdir,
+        query_fp = '%s/seqs.fna' % str(query)
```

Comment:
Did you run into problems using `QIIME1DemuxFormat`? This approach is fragile (what if the dirfmt changes the file name from `seqs.fna`?), you can shore it up using:

```python
query_fp = str(query.file.view(QIIME1DemuxFormat))
```

But, I think just using the `QIIME1DemuxFormat` would be preferable (unless you ran into an issue, in which case let us know!).

Reply:
Yeah, transformers don't exist going to `QIIME1DemuxFormat`, only to `QIIME1DemuxDirFmt` ([eg](https://github.com/qiime2/q2-types/blob/261cb96f2c7af2c671e6e1a1011f0502b7604817/q2_types/per_sample_sequences/_transformer.py#L360-L387) - I had missed that before).

Reply:
Oh yeah, that makes sense. Can you make the [change I suggested above](https://github.com/qiime2/q2-shogun/pull/15#discussion_r392316258), in that case?

Reply:
Will do, thanks!